src/components/icons/UnderlineIcon.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import Icon from '../../@types/Icon'
+
+// eslint-disable-next-line jsdoc/require-jsdoc
+const UnderlineIcon = ({ size = 20, style }: Icon) => (
```

Comment:
Currently the "U" is much larger than the "B" and "I". The "U" should match the size of "B" and "I" for visual consistency.

Reply:
I've changed the UnderlineIcon. I tried to match the "B", "I" and "U" icons.